packages/components/src/combobox-control/index.tsx

```diff
@@ -279,13 +351,14 @@ function ComboboxControl( {
 					{ isExpanded && (
 						<SuggestionsList
 							instanceId={ instanceId }
-							match={ { label: inputValue } }
+							match={ { label: inputValue, value: '' } }
```

Comment:
The upstream type in `SuggestionsList` here is technically a little too strict than it needs to be for `match` itself, but on the whole it is an understandable compromise because the `displayTransform` prop used to process `match` values also needs to handle `suggestion` values. So they kind of should be the same shape, at least for simplicity's sake.

Should not cause any changes in runtime behavior.

Reply:
Thank you for the explanation. I'd love to simplify this in the future, but your solution for now is definitely the right choice.

Do you think there would be value in adding a comment explaining the empty string for `value` ?